smurfs/src/reducers/index.js

```diff
@@ -1,18 +1,23 @@
 /*
   Be sure to import in all of the action types from `../actions`
 */
+import {
+  FETCH_START, 
+  FETCH_SUCCESS, 
+  FETCH_ERROR, 
+  ADD_SMURF, 
+  POST_SUCCESS
+} from '../actions';
 
-/*
- Your initial/default state for this project could *Although does not have to* look a lot like this
- {
+ const initialState = {
    smurfs: [],
-   fetchingSmurfs: false
-   addingSmurf: false
-   updatingSmurf: false
-   deletingSmurf: false
+   fetchingSmurfs: false,
```

Comment:
You have different pieces od information on your initial state that you are not using, this won't break your code and is not part of the MVP but you can clean this reducer a bit.